remove extra space at top of modules

elstoc opened the pull request:
where a lib or iop module is split into sections, reduce the amount of
space added at the top of the first section.

impacts splittoning, watermark and export modules

johnny-bit commented:
Can you add comment/documentation somewhere about it? I use sections in my temperature.c PR and I'd probably miss this stuff :/

elstoc commented:
> Can you add comment/documentation somewhere about it?

Where would you like it added? This is something I introduced when I split the filmic reconstruct tab into sections and it looked silly with a big gap at the top, so I've just reapplied it to some more modules. Just add the class section_label_top to the top section label and it will get a reduced top margin.
